docs/build-system.md

```diff
@@ -86,7 +86,7 @@ opam install reason
 Create a file named `dune-project`. This file will tell Dune a few things about
 our project configuration:
 
-```text
+```dune
```

Comment:
Technically this is a `dune-project` file, but not sure it's worth it to add https://github.com/ocamllabs/vscode-ocaml-platform/blob/master/syntaxes/dune-project.json

Reply:
I think it's ok as is.